google-cloud-core/src/main/java/com/google/cloud/GrpcServiceOptions.java

```diff
@@ -226,7 +226,7 @@ protected GrpcServiceOptions(
   /**
    * Returns a builder for API call settings.
    */
-  protected ApiCallSettings.Builder apiCallSettings() {
+  protected UnaryApiCallSettings.Builder apiCallSettings() {
```

Comment:
Yeah this is inconsistent. @shinfan , could you address the inconsistency? Maybe removing the `Api` from the names would be best. We needed `Api` on `ApiCallable` because using `Callable` on its own would be confusing due to the existing type in Java, but the other names don't have that problem. 

@mziccard it's not clear which class you are referencing when you say "Should we consider renaming it?" I can see how `PageStreamingCallSettings` might seem more related to `StreamingCallSettings`. Maybe we should call the first one `PagedListCallSettings`.

Does this scheme work?
* `UnaryCallable` and `UnaryCallSettings`
  * `PagedListCallSettings`
* `StreamingCallable` and `StreamingCallSettings`